plugins/prefix/plugin.go

```diff
@@ -77,8 +77,8 @@ type lease struct {
 	Expire time.Time
 }
 
-// Handler holds state of allocations for the plugin
-type Handler struct {
+// PluginState holds state of allocations for the plugin
```

Comment:
If we're changing this to make it homogeneous across plugins, let's also make it private at the same time

Reply:
Allready done